docs/topics/clusterdefinitions.md

```diff
@@ -269,7 +269,7 @@ Below is a list of kubelet options that aks-engine will configure by default:
 | "--image-gc-low-threshold"          | "850"                                                                                                                                                         |
 | "--non-masquerade-cidr"             | "10.0.0.0/8"                                                                                                                                                  |
 | "--azure-container-registry-config" | "/etc/kubernetes/azure.json"                                                                                                                                  |
-| "--pod-max-pids"                    | "100" (need to activate the feature in --feature-gates=SupportPodPidsLimit=true)                                                                              |
+| "--pod-max-pids"                    | "-1" (need to activate the feature in --feature-gates=SupportPodPidsLimit=true)                                                                              |
```

Comment:
We should track an issue to make this a sensible default as a follow up item (-1 doesn't really make sense as a default value).

Reply:
I agree, but that's what the work-in-progress docs say: https://github.com/kubernetes/website/pull/13806

Reply:
I also asked for clarification on what `0` means. If it disables the feature, then I think we may actually want `0` instead of `-1` (node enforceable limit) if we don't set a node enforceable limit today.